src/sqlfluff/core/parser/segments/base.py

```diff
@@ -140,12 +140,12 @@ def __repr__(self):
     @property
     def _comments(self):
         """Returns only the comment elements of this segment."""
-        return [seg for seg in self.segments if seg.type == "comment"]
```

Comment:
I see that this PR involves several changes like this, going from `type ==` to `is_type()`. Is the former style no longer allowed? If so, have we (or should) we try and systematically find and update all occurrences of this usage?

Reply:
I have done exactly that 😄